lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_handlers.c

```diff
@@ -662,6 +665,7 @@ void sgw_s8_handle_modify_bearer_request(
         pgw_ipv6 = &bearer_ctx_p->p_gw_address_in_use_up.address.ipv6_address;
       }
 
+#if !MME_UNIT_TEST
```

Comment:
any reason why this is not moved to just before `gtp_tunnel_ops->send_end_marker(enb, modify_bearer_pP->teid);`

Reply:
I put it here because after validating the condition, ue_ip is fetched and  ovs related functions, 
send_end_marker() and gtpv1u_del_s8_tunnel() are called. 
For executing unit test case, we can avoid validating the check and calling ovs related functions()